src/client/CodeEditor/InteractiveWidgets/colorsInTextPlugin.ts

```diff
@@ -72,8 +73,42 @@ class ColorWidget extends WidgetType {
     svg.appendChild(colorCircle);
 
     parent.appendChild(svg);
+
+    // Hovering tool tip shortcut for when users hover over color circle
+    const hoveringTooltip = document.createElement('div');
```

Comment:
Sounds good. I'll look into it and change it to use the existing infrastructure then.

Reply:
Oh sorry, my bad - I didn't realize this file was in a different realm, so to speak, since it's not even using React for rendering. This is part of the CodeMirror plugin! Got it... In that case I think your solution is very nice.